README.md

```diff
@@ -15,7 +15,7 @@ storage foundation for a Free Internet!
 If you have [Go](https://golang.org/cmd/go/) installed you can run:
 
 ```
-go get -u github.com/NebulousLabs/skynet-cli...
+go get -u github.com/NebulousLabs/skynet-cli/cmd...
```

Comment:
I think you just need the `/`

```suggestion
go get -u github.com/NebulousLabs/skynet-cli/...
```

Reply:
I confirmed deleting the binary and deleting the skynet-cli repo that this command works.

Reply:
I confirmed it as well.